docs/src/development/environment-setup.md

```diff
@@ -13,39 +14,35 @@ If you already have Rust installed make sure to upgrade it by running
 $ rustup update
 ```
 
-To verify that project was cloned and set up correctly, you can run
+### Scarb
+You can read more about installing scarb [here](https://docs.swmansion.com/scarb/download.html)
 
-```shell
-$ cargo check
-```
+Please make sure you're using scarb installed via asdf - otherwise some tests may fail.
+> To verify, run:
+> 
+> ```shell
+> $ which scarb
+> $HOME/.asdf/shims/scarb
+> ```
+> 
+> If you previously installed scarb using official installer, you may need to remove this installation or modify your PATH to make sure asdf installed one is always used.
```

Comment:
This doesn't really tell why it's necessary

Reply:
Discussed offline, not necessary and too complicated to explain with little value.